pages/linux/pacman4console.md

```diff
@@ -0,0 +1,21 @@
+# pacman4console
+
+> A text-based console game inspired by the original Pacman.
+> You play as Pacman (the `C`), avoid the ghosts (the `&`s), and collect all the pellets (the `*`s and `.`s).
```

Comment:
Do we really need to have this second line of description? It seems overly specific and kind of unnecessary to me. :thinking:

Reply:
Well, seeing as the game *is* based on Pacman, and the characters used to represent Pacman, the ghosts, and the pellets are about as similar to their original forms as possible, I could see how this could be spurious.

Reply:
Yep, I think the game is already intuitive enough, no need to explain it.

Reply:
Alright; point taken (since with the addition of the repo's link, people could look it up and get that information there).